src/app/jscaip/Coord.ts

```diff
@@ -143,6 +161,18 @@ export class Coord extends Vector {
         }
         return coords;
     }
+
+    public getAllCoordsToward(end: Coord): Coord[] {
+        if (this.equals(end)) {
+            return [end];
+        } else {
+            // eslint-disable-next-line @typescript-eslint/no-this-alias
```

Comment:
Il est toujours nécessaire ce disable ? Car on alias pas this directement, mais on le met dans un tableau. Si oui, feel free to resolve direct !